packages/service/lib/stackable.js

```diff
@@ -256,6 +256,24 @@ class ServiceStackable {
       getCustomLabels: req => {
         const telemetryId = req.headers['x-plt-telemetry-id'] ?? 'unknown'
         return { telemetry_id: telemetryId }
+      },
+      summary: {
```

Comment:
Why do you need this?. `@platformatic/fastify-http-metrics` registers fastify hooks and collect metrics.
https://github.com/platformatic/fastify-http-metrics/blob/dc34411d378da0f121dc3bf543fcc93b63b9e69c/index.js#L48

Reply:
So this is needed to set `0` as an init value, right?